src/HotChocolate/Directory.Build.props

```diff
@@ -4,6 +4,6 @@
     <PackageIcon>hotchocolate-signet.png</PackageIcon>
   </PropertyGroup>
   <PropertyGroup>
-    <BananaCakePopVersion>16.0.1</BananaCakePopVersion>
+    <NitroVersion>17.0.1-insider.4</NitroVersion>
```

Comment:
Should we move this to package management?

Reply:
CPM?

`Build.Nitro.cs` does:

```cs
project.SetProperty("NitroVersion", latestVersion);
```

... not sure if that can be changed?

Reply:
Ah ok ... lets leave it.